inc/spbc-settings.php

```diff
@@ -2519,7 +2519,8 @@ function spbc_field_traffic_control_log()
         echo '</div>';
         return;
     }
-
+    $autoblock_amount = isset($spbc->settings['traffic_control__autoblock_amount']) ? $spbc->settings['traffic_control__autoblock_amount'] : 1000;
+    $autoblock_timeframe = isset($spbc->settings['traffic_control__autoblock_timeframe']) ? (int)$spbc->settings['traffic_control__autoblock_timeframe'] / 60 : 5;
```

Comment:
Please, use `$spbct->default_settings['traffic_control__autoblock_timeframe']` instead of hardcoded value.

Reply:
ok